include/ensmallen_bits/indicators/igd.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file igd.hpp
+ * @author Rahul Ganesh Prabhu
+ *
+ * Inverse Generational Distance (IGD) indicator.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_INDICATORS_IGD_HPP
+#define ENSMALLEN_INDICATORS_IGD_HPP
+
```

Comment:
It might be a good idea to cite: "Scalable Multi-Objective Optimization Test Problems". And add a simple class description for each function, similar to: https://github.com/mlpack/ensmallen/blob/master/include/ensmallen_bits/problems/himmelblau_function.hpp

Reply:
Yup, I'll do the documentation once I finish the code for reference points